src/nunit.analyzers/ConstActualValueUsage/ConstActualValueUsageCodeFix.cs

```diff
@@ -0,0 +1,102 @@
+            var invocationSyntax = argumentSyntax.Ancestors()
+                .OfType<InvocationExpressionSyntax>()
+                .FirstOrDefault();
+
+            if (invocationSyntax == null)
+                return;
+
+            if (!TryFindArguments(semanticModel, invocationSyntax,
+                out var expectedArgument,
+                out var actualArgument))
+            {
+                return;
+            }
+
+            var newRoot = root
+                .ReplaceNodes(new[] { expectedArgument, actualArgument },
+                    (node, _) => node == actualArgument ? expectedArgument : actualArgument);
+
+            context.RegisterCodeFix(
+                CodeAction.Create(
+                    CodeFixConstants.SwapArgumentsDescription,
+                    _ => Task.FromResult(context.Document.WithSyntaxRoot(newRoot))), context.Diagnostics);
+        }
+
+        private static bool TryFindArguments(SemanticModel semanticModel, InvocationExpressionSyntax invocationSyntax,
+            out ExpressionSyntax expectedArgument, out ExpressionSyntax actualArgument)
+        {
+            expectedArgument = null;
+            actualArgument = null;
+
+            var methodSymbol = semanticModel.GetSymbolInfo(invocationSyntax).Symbol as IMethodSymbol;
+
+            if (methodSymbol == null)
+                return false;
+
+            // option 1: Assert with 'expected' and 'actual' parameters (e.g. Assert.AreEqual(expected, actual) )
+            if (methodSymbol.Parameters.Length >= 2
+                && methodSymbol.Parameters[0].Name == NunitFrameworkConstants.NameOfExpectedParameter
+                && methodSymbol.Parameters[1].Name == NunitFrameworkConstants.NameOfActualParameter)
+            {
+                expectedArgument = invocationSyntax.ArgumentList.Arguments[0].Expression;
+                actualArgument = invocationSyntax.ArgumentList.Arguments[1].Expression;
+                return true;
+            }
+
+            // option 2: Assert with 'actual' and 'constraint' parameters, and provided constraint has 'expected' parameter
+            // (e.g. Assert.That(actual, Is.EqualTo(expected)))
+            if (methodSymbol.Parameters.Length >= 2
+                && methodSymbol.Parameters[0].Name == NunitFrameworkConstants.NameOfActualParameter
+                && methodSymbol.Parameters[1].Name == NunitFrameworkConstants.NameOfExpressionParameter)
+            {
+                actualArgument = invocationSyntax.ArgumentList.Arguments[0].Expression;
+
+                var constraintExpression = invocationSyntax.ArgumentList.Arguments[1].Expression as InvocationExpressionSyntax;
+
+                if (constraintExpression == null)
+                    return false;
+
+                var constraintMethod = semanticModel.GetSymbolInfo(constraintExpression).Symbol as IMethodSymbol;
+
+                if (constraintMethod == null)
+                    return false;
+
```

Comment:
I think that we should return here if `constraintExpression` does not match `Is.____(expected)` for some method name `____`.

Reply:
Do you think we shouldn't support cases like `Assert.That(smth, Is.Not.EqualTo(123))`?

Reply:
I think it is okay also to support this if it does not make the code too complex. As far as I can tell `Is` has the following methods that are symmetric `EqualTo`, `SameAs`, and `SamePath`, and each of these can be "prefixed" by zero or more `.Not` (but I think it is ok that the code only handles zero or one).